### PR TITLE
Content styling

### PR DESCRIPTION
**Pull request by tatianainama**
This PR applies design styles to markdown components (notes/admonitions, headings, etc)
~~Still trying to figure it out how to change admonitions' icons without swizzling the component. Will update those in another PR~~

### CODE REVIEW

File: docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -48,6 +51,32 @@ const defaultPlugins = SDK_FOLDERS.map((SDK) => {
             sidebarPath: require.resolve(sidebarPath),
           }
         : {}),
+      admonitions: {
+        infima: true,
+        customTypes: {
+          note: {
+            ifmClass: 'note',
+            svg: '<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7 13.7A6.7 6.7 0 117 .3a6.7 6.7 0 010 13.4zm0-1.4A5.3 5.3 0 107 1.7a5.3 5.3 0 100 10.6zm-.7-8.6h1.4V5H6.3V3.7zm0 2.6h1.4v4H6.3v-4z" fill="#000" fill-opacity=".4"/></svg>'
```
Comment (vishalnarkhede) [inline docusaurus/docusaurus.config.js:59]: Can we move svg content to separate files somehow? And import then in docusaurus.config.js

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -30,6 +30,9 @@ const CUSTOM_PLUGINS = CUSTOM_PLUGIN_FILES.map((file) => {
   return sdkConfig.plugins;
 }).flat();
 
+const CUSTOM_CSS_PATH = path.join(__dirname, 'src/css');
+const CUSTOM_CSS_FILES = fs.readdirSync(CUSTOM_CSS_PATH).map(file => `${CUSTOM_CSS_PATH}/${file}`);
```
Comment (tsirlucas) [inline docusaurus/docusaurus.config.js:34]: noice

File: docusaurus/src/theme/Heading/styles.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// overwriting docusaurus theme classic styles
+h1.docTitle_node_modules-\@docusaurus-theme-classic-lib-next-theme-DocItem- {
```
Comment (tsirlucas) [inline docusaurus/src/theme/Heading/styles.scss:9]: hey, do you think we can find another way to change that? this `node_modules` thing makes me feel like its auto generated. maybe once the heading is swizzled we can manually add another one?

what exactly is this selector selecting? if its the title of the page, in stg that is different: 

image

**Review by tsirlucas — CHANGES_REQUESTED** [review]
agree with vishs comment and added another one

please ignore the "noice" one

File: docusaurus/src/theme/Heading/styles.scss (hunk 3 above)
Comment (tatianainama) [reply docusaurus/src/theme/Heading/styles.scss:9]: Yeah, it was supposed to select the page title, I didn't realise it was randomly generated 🤦  I'll see if I can select it differently or swizzle it if necessary

File: docusaurus/docusaurus.config.js (hunk 1 above)
Comment (tatianainama) [reply docusaurus/docusaurus.config.js]: > Can we move svg content to separate files somehow? And import then in docusaurus.config.js

@vishalnarkhede @tsirlucas The admonition component only accepts inline svgs strings (no file reference like 'img/icon.svg') so I decided to move those into another file and reference them _as strings_ in the config, let me know what you think of this solution

**Review by tsirlucas — APPROVED** [review]

**Pull request merged**